src/shared/ui/paragraph/paragraph.test.tsx

```diff
@@ -1,33 +1,70 @@
 import { render, screen } from '@testing-library/react';
 import { describe, expect, it } from 'vitest';
-import { Paragraph } from './paragraph';
+import { Paragraph, cx } from './paragraph';
 
 describe('Paragraph component', () => {
-  it('renders children text correctly', () => {
-    render(<Paragraph>Test Text</Paragraph>);
+  it('renders without crashing', () => {
+    render(<Paragraph />);
+    const paragraph = screen.getByTestId('paragraph');
 
-    const paragraphElement = screen.getByText('Test Text');
+    expect(paragraph).toBeVisible();
```

Comment:
Let's move it on 23 line and delete this test-case

Reply:
[Resolved](https://github.com/rolling-scopes/site/pull/475/commits/cee2141deb51a6905a4ab1bb969ea1a07116819d).